swmm-toolkit/src/swmm/toolkit/solver.i

```diff
@@ -291,6 +158,26 @@
 }
 
 
+/* TYPEMAPS FOR ENUMERATED TYPE OUTPUT ARGUMENTS */
```

Comment:
@michaeltryby Can you explain this function to me? It looks cool, but I am having trouble understanding module and function. Is this function replacing the swig stats function?

Reply:
@jennwuu Yeah this typemap is kind of cool. What it does is it goes in and instantiates an instance of the enum class based in the argument type and value being returned. This make it really nice to work with enum types in the resulting python code cause you get a python enum like you would expect instead of an integer value.